packages/service-core/src/storage/MongoBucketStorage.ts

```diff
@@ -445,7 +457,8 @@ export class MongoBucketStorage
           operationType: 1,
           'fullDocument._id': 1,
           'fullDocument.last_checkpoint': 1,
-          'fullDocument.last_checkpoint_lsn': 1
+          'fullDocument.last_checkpoint_lsn': 1,
+          'fullDocument.content': 1
```

Comment:
This has the potential to add a lot of overhead - the rest of the change stream is optimized to get checkpoint updates with as little latency as feasible. It also shouldn't be needed - the content of a sync rules document never change.